src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpDelete.java

```diff
@@ -53,6 +58,25 @@ public void ldpcvThatAdvertisesDeleteShouldRemoveContainerAndMementos(final Stri
                                         "https://fcrepo.github.io/fcrepo-specification/#ldpcv-delete",
                                         ps);
 
+        //create a versioned resource
+        final Response response = createVersionedResource(uri, info);
+        final String versionedResource = getLocation(response);
+        final String timeMap = getTimeMapUri(response).toString();
+
+        final Response optionsResponse = doOptions(timeMap);
+
+        //check that delete is supported
+        if (hasHeaderValueInMultiValueHeader("Allow", "DELETE", optionsResponse)) {
+            //delete the ldpcv
+            final Response deleteResponse = doDelete(timeMap);
+            assertEquals("Timemap DELETE command failed.", deleteResponse.getStatusCode(), 204);
+            //verify that ldpcv is gone
+            final Response timeMapGet = doGetUnverified(timeMap);
+            assertEquals("Timemap was not deleted", timeMapGet.getStatusCode(), 404);
+            //verify that versioning model is gone
+            final Response getResponse = doGet(versionedResource);
+            confirmAbsenceOfLinkValue(ORIGINAL_RESOURCE_LINK_HEADER, getResponse);
```

Comment:
We should probably have an `else` block that adds a "Not Supported" result to the report.
For now, a `TODO: <new GitHub issue>` may be sufficient.

Reply:
@awoods : Are you sure we want to do this?  It seems to me that perhaps there is a missing "MAY" test (as in "LDPCv may support DELETE" that will fail if delete is not supported.  I believe that is how were are currently doing it with other MAY tests.